Dockerfile

```diff
@@ -1,61 +1,61 @@
+    bash \
+    build-base \
+    curl \
+    ca-certificates \
     graphviz \
-    rubygem-nokogiri \
+    jpeg \
+    jpeg-dev \
+    patch \
+    python2 \
+    python2-dev \
+    py-pip \
+    ruby \
+    ruby-dev \
+    tar \
+    ttf-liberation \
     unzip \
     findutils \
     which \
     wget \
-    python-devel \
-    zlib-devel \
-    libjpeg-devel \
-    redhat-rpm-config \
-    patch \
-    sudo \
-  && dnf clean packages \
-  && (curl -s -k -L -C - -b "oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u112-b15/jdk-8u112-linux-x64.tar.gz | tar xfz -) \
-  && mkdir /fopub \
-  && curl -L https://api.github.com/repos/asciidoctor/asciidoctor-fopub/tarball | tar xzf - -C /fopub/ --strip-components=1 \
-  && touch /tmp/empty.xml \
-  && fopub /tmp/empty.xml \
-  && rm /tmp/empty.xml \
-  && gem install --no-ri --no-rdoc asciidoctor --version $ASCIIDOCTOR_VERSION \
+    zip \
+    zlib-dev \
   && gem install --no-ri --no-rdoc asciidoctor-diagram \
   && gem install --no-ri --no-rdoc asciidoctor-epub3 --version 1.5.0.alpha.6 \
+  && gem install --no-ri --no-rdoc asciidoctor-revealjs \
   && gem install --no-ri --no-rdoc rake \
   && gem install --no-ri --no-rdoc epubcheck --version 3.0.1 \
-  && gem install --no-ri --no-rdoc kindlegen --version 3.0.1 \
-  && gem install --no-ri --no-rdoc asciidoctor-pdf --version 1.5.0.alpha.13 \
+  && gem install --no-ri --no-rdoc kindlegen --version 3.0.3 \
+  && gem install --no-ri --no-rdoc prawn --version 2.1.0 \
+  && gem install --no-ri --no-rdoc asciidoctor-pdf --version 1.5.0.alpha.14 \
   && gem install --no-ri --no-rdoc asciidoctor-confluence \
-  && gem install --no-ri --no-rdoc rouge coderay pygments.rb thread_safe epubcheck kindlegen \
-  && gem install --no-ri --no-rdoc slim \
-  && gem install --no-ri --no-rdoc haml tilt \
-  && gem install --no-ri --no-rdoc asciidoctor-revealjs \
-  && mkdir $BACKENDS \
-  && (curl -LkSs https://api.github.com/repos/asciidoctor/asciidoctor-backends/tarball | tar xfz - -C $BACKENDS --strip-components=1) \
-  && wget https://bitbucket.org/pypa/setuptools/raw/bootstrap/ez_setup.py -O - | python \
-  && easy_install "blockdiag[pdf]" \
-  && easy_install seqdiag \
-  && easy_install actdiag \
-  && easy_install nwdiag \
+  && gem install --no-ri --no-rdoc rouge coderay pygments.rb thread_safe \
+  && gem install --no-ri --no-rdoc slim haml tilt \
+  && pip install --no-cache-dir --upgrade pip \
+  && pip install --no-cache-dir 'blockdiag[pdf]' \
+  && pip install --no-cache-dir seqdiag \
+  && pip install --no-cache-dir actdiag \
+  && pip install --no-cache-dir nwdiag \
   && (curl -s get.sdkman.io | bash) \
-  && /bin/bash -c "source /root/.sdkman/bin/sdkman-init.sh" \
-  && /bin/bash -c "echo sdkman_auto_answer=true > ~/.sdkman/etc/config" \
-  && /bin/bash -c -l "sdk install lazybones"
+  && bash -c "source /root/.sdkman/bin/sdkman-init.sh" \
+  && bash -c "echo sdkman_auto_answer=true > ~/.sdkman/etc/config" \
```

Comment:
I just removed lazybone following this @mojavelinux comment: https://github.com/asciidoctor/docker-asciidoctor/pull/39#issuecomment-288642823 

But you are right, sdkman does not seems needed now, I also don't see so much value to install it, giving the maintenance overhead. I'm removing it for now, I can still change that later.